src/IssueViz.Security/Taint/TaintIssuesSynchronizer.cs

```diff
@@ -49,18 +51,26 @@ internal sealed class TaintIssuesSynchronizer : ITaintIssuesSynchronizer
         private readonly IConfigurationProvider configurationProvider;
         private readonly ILogger logger;
 
+        private readonly VSShellInterop.IVsMonitorSelection vsMonitorSelection;
```

Comment:
The product change is relatively simple; the taint synchronizer sets/clears the UIContext when the binding changes. The context is only set if taint issues exist (so it won`t ever be set for SonarQube Community edition).